docs/users-guide/Defining-Components.adoc

```diff
@@ -932,6 +932,22 @@ What do you think will happen if you run `fpp-check`
 on this code?
 Try it and see.
 
+==== Interfaces
```

Comment:
Let's rename this section Imported Interfaces. It should describe how to import interfaces into a component definition.